tests/Rule/EqualTest.php

```diff
@@ -4,18 +4,22 @@
 
 namespace Yiisoft\Validator\Tests\Rule;
 
+use RuntimeException;
 use Yiisoft\Validator\Rule\Equal;
-use Yiisoft\Validator\SerializableRuleInterface;
+use Yiisoft\Validator\Tests\Rule\Base\RuleTestCase;
+use Yiisoft\Validator\Tests\Rule\Base\SerializableRuleTestTrait;
 
-final class EqualTest extends AbstractRuleTest
+final class EqualTest extends RuleTestCase
 {
+    use SerializableRuleTestTrait;
```

Comment:
Seems like "different rule in handler" test skipped here?

Reply:
No. It's not need, because in this test and in some others used one handler `CompareHandler`.

Reply:
:ok_hand: